monitoring/grafana/provisioning/dashboards/court-backend.json

```diff
@@ -253,5 +246,5 @@
   "variables": {
     "list": []
   },
-  "version": 1
+  "version": 2
```

Comment:
should we update the version number every time we change this file? grafana wouldn't update it otherwise?

Reply:
When you edit the dashboard and click save -> export JSON this number is generated automatically.